neutromeratio/restraints.py

```diff
@@ -34,23 +77,23 @@ def __init__(self, sigma:unit.Quantity, atom_i_idx:int, atom_j_idx:int, atoms:st
         e : float
             bias
         """
-
-        assert(type(sigma) == unit.Quantity)
-        k = (kB * temperature) / (sigma**2)
-        self.k = k.value_in_unit((unit.kilo * unit.joule) / ((unit.angstrom **2) * unit.mole))
-        self.device = device
+        super().__init__(sigma, active_at_lambda)
         self.atom_i_element = atoms[atom_i_idx]
         self.atom_j_element = atoms[atom_j_idx]
         self.atom_i_idx = atom_i_idx
         self.atom_j_idx = atom_j_idx
         self.mean_bond_length = (bond_length_dict[frozenset([self.atom_i_element, self.atom_j_element])]).value_in_unit(unit.angstrom)
-        self.active_at_lambda = active_at_lambda
         self.upper_bound = self.mean_bond_length + 0.2
```

Comment:
Leave a `# TODO: ` comment here noting that the width of the restraint is hardcoded